_posts/2018-08-31-foreman-community-newsletter---august-2018.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Foreman Community Newsletter - August 2018
+date: 2018-08-31 15:17
+author:
+  name: Greg Sutcliffe
+  uri: gwmngilfen
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  New releases! Oh, so many releases....
+---
+
+### Releases for everyone!
+
+We've had a lot of releases this month, so I'll group them all together. We've released:
+
+* [Foreman 1.19.0][1_19_0] was released on the very last day of August,
+  featuring support for Ubuntu Bionic, syslog logging, reporting failed builds,
+  and more ### Foreman 1.18.0 released
+* [1.17.3 and 1.17.4][1_17_4] bug fix releases. If you are still on 1.17, be
+  sure to upgrade to 1.17.4 before upgrading to 1.18!
+* [Katello 3.8 RC 2 & 3][k_3_8] - do give the RC a try and let us know how you find it :)
+
+With the release of 1.19.0, this marks 1.17.X as now unmaintained - 1.17.4 will
+be the last release in the 1.17 series. Please do upgrade as soon as you can.
+
+As ever, many thanks to the contributors to these releases - coders, RC testers,
+documentation writers, and bug/feature reporters. Foreman wouldn't be what it
+is without you! If you do find issues with, please do [report them!][issues]
+
+Big thanks also to Ondrej, Ewoud, and Tomer for their effort in bringing these
+releases out at this rate, there's been a *lot* of work behind the scenes.
+
+### New RFCs for container work
+
+In the [survey analysis]() I posted earlier this year, I called attention to
+the idea of supporting Foreman being run in a container, and that more would
+come on this later. That time is now :)
+
+Eric has spent a lot of time working through the implications of this, and has
+now posted a set of RFCs to the Development board for review. There's a *lot*
+to read here, so it's broken down into multiple posts. You can get the
+top-level over on [this forum
+post](https://community.theforeman.org/t/containerizing-the-foreman-ecosystem/10948)
+and then dive into details from there.
+
+I'd recommend for *anyone* with an interest in the future direction of Foreman
+to read over these, and give their thoughts on any part of it which interests
+you. Thanks!
+
+### Foreman at FrosCon
```

Comment:
The name of the conference is "FrOSCon".

Reply:
Fixed, ty